rest/perf_routes.go

```diff
@@ -204,15 +204,16 @@ func (h *perfExistsByTaskIdHandler) Run(ctx context.Context) gimlet.Responder {
 		}))
 		return gimlet.MakeJSONErrorResponder(err)
 	}
-	if len(perfResults) > 0 {
-		existResult := model.APIPerformanceResultExists{
-			Name:   h.opts.TaskID,
-			Exists: true,
-		}
-		return gimlet.NewJSONResponse(existResult)
+	if len(perfResults) == 0 {
```

Comment:
The way this is written to return the number of results, it seems like 0 is just a special case and not necessarily an error. Would it be more reasonable to just return no perf results in the same way as non-zero perf results and let the client (in this case, Evergreen) deal with the 0 results case however it wants?

Reply:
Agreed, that would be more general